cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +558,36 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
+            args:
+            - skipper
+            - -address=:8443
+            - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+            - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+            - -insecure
```

Comment:
@szuecs @mikkeloscar Do you mind having `-insecure` here?

Otherwise we'd need to pass the `ca` to skipper. I tried with comma-separating `-tls-cert` (like the docs say) but skipper failed. Should that work?

Reply:
Is it complicated to pass the ca? It's already on the node. :)

Reply:
You probably can also cat ca.pem and apiserver.pem into a apiserver-combined.pem, which is very common setup in webservers. On the other hand localhost is also fine to ignore, because if you can wiretap localhost you most likely can also read the apiserver-key.pem....
If we need to change the CA, and the workaround above does not work we need a little PR to skipper to be able to replace the RootCAs.

Reply:
Alright, then I suggest we keep `insecure`.